content/events/Zama-afterwork-party.md

```diff
@@ -0,0 +1,40 @@
+---
+eventName: "Zama Afterwork Party @The CoFHE Shop" #the name of your track or event and its mandatory
+
+eventDescription: "​Join Zama and the FHE community in a dedicated space for all things Fully Homomorphic Encryption (FHE) during Ethcc [7]. Network, hack, drink good coffee." #short description of your track or event limiting to 100-150 characters
```

Comment:
@laeccles Please confirm if it's 'EthCC [7] week' or just 'EthCC week'.

Reply:
@WinsCodes Please remove [7].

Thank you!